Makefile

```diff
@@ -26,23 +30,26 @@ dist/tool-install: Makefile
 	mkdir -p dist
 	touch dist/tool-install
 
+.PHONY: dist/buf
 dist/buf: dist/buf-go dist/buf-web
 
 dist/buf-go: dist/tool-install $(PROTO_FILES)
-	PATH=$(PATH_WITH_TOOLS) buf lint
-	PATH=$(PATH_WITH_TOOLS) buf format -w
-	PATH=$(PATH_WITH_TOOLS) buf generate --template ./proto/viam/buf.gen.yaml
-	PATH=$(PATH_WITH_TOOLS) buf generate --template ./proto/viam/buf.gen.tagger.yaml
+	buf lint
+	buf format -w
+	buf generate --template ./proto/viam/buf.gen.yaml
+	buf generate --template ./proto/viam/buf.gen.tagger.yaml
 	touch dist/buf-go
 
 dist/buf-web: dist/tool-install $(PROTO_FILES)
-	PATH=$(PATH_WITH_TOOLS) buf lint
-	PATH=$(PATH_WITH_TOOLS) buf format -w
-	PATH=$(PATH_WITH_TOOLS) buf generate --template ./proto/viam/buf.gen.web.yaml
-	PATH=$(PATH_WITH_TOOLS) buf generate --timeout 5m --template ./proto/viam/buf.gen.web.yaml buf.build/googleapis/googleapis
+	buf lint
+	buf format -w
+	buf generate --template ./proto/viam/buf.gen.web.yaml
+	buf generate --timeout 5m --template ./proto/viam/buf.gen.web.yaml buf.build/googleapis/googleapis
+	touch dist/buf-web
 
+.PHONY: lint
 lint: dist/tool-install
-	PATH=$(PATH_WITH_TOOLS) buf lint
-	PATH=$(PATH_WITH_TOOLS) buf format -w
-	export pkgs=`go list -f '{{.Dir}}' ./... | grep -v gen | grep -v proto` && echo "$$pkgs" | xargs go vet -vettool=$(TOOL_BIN)/combined
-	export GOGC=50 pkgs=`go list -f '{{.Dir}}' ./... | grep -v gen | grep -v proto` && echo "$$pkgs" | xargs $(TOOL_BIN)/golangci-lint run $(LINT_BUILD_TAGS) -v --fix --config=./etc/.golangci.yaml
+	buf lint
+	buf format -w
+	export pkgs=`go list -f '{{.Dir}}' ./... | grep -v gen | grep -v proto` && echo "$$pkgs" | xargs go vet -vettool=`which combined`
```

Comment:
`-vettool=combined` produced the following error

```shell
export pkgs=`go list -f '{{.Dir}}' ./... | grep -v gen | grep -v proto` && echo "$pkgs" | xargs go vet -vettool=combined
go: can't execute /Users/mc/viam/api/combined -flags: fork/exec /Users/mc/viam/api/combined: no such file or directory
```

I decided to go with `` `which combined` `` to enable the requested removal of the `TOOL_BIN` variable, going off an example from the `go vet` docs. Also happy to bring `TOOL_BIN` back into the mix, if that's preferable

```shell
$ go help vet
...
The -vettool=prog flag selects a different analysis tool with alternative
or additional checks.
For example, the 'shadow' analyzer can be built and run using these commands:

  go install golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
  go vet -vettool=$(which shadow)
```